feeadjuster/feeadjuster.py

```diff
@@ -102,12 +108,33 @@ def forward_event(plugin: Plugin, forward_event: dict, **kwargs):
             plugin.log("Adjusting fees: " + str(e), level="error")
 
 
+@plugin.method("forcefeeadjust")
```

Comment:
also, we might think about renaming this to just `feeadjust` as long as its the only command from the plugin starting with the name of the plugin makes it a bit easier to know which command belongs to which plugin.

Reply:
Also, do we want a testcase for this plugin method by i.e. setting up a small mesh with one node in the middle and three around each with different fees and expected fees. Calling this method on the middle node should update the three channels...